packages/icons/src/index.ts

```diff
@@ -24,7 +24,8 @@ if (typeof window !== 'undefined') {
   }
 
   // Defining clr-icon custom element
-  customElements.define('clr-icon', ClarityIconElement);
+  const clarityIconElementFn: unknown = ClarityIconElement;
```

Comment:
What happened here?

Reply:
I reverted for now. Updating the whole project to TS 3.9 made eslint complain about this. I could also use ts-ignore here.